ocdaction/ocdaction/templates/challenge/challenge_list.html

```diff
@@ -10,12 +10,32 @@
         </a>
         {% if challenges %}
             {% for challenge in challenges %}
+                {% if challenge.in_progress %}
                 <ul class="col-xs-6 col-xs-offset-3">
+                    <h4>In progress</h4>
                     <li class="row">
                         <a href="{% url 'challenge' challenge.id %}"><h3>{{ challenge.challenge_name }}</h3></a>
-                        <a class="btn btn-primary pull-right col-xs-3" href="{% url 'challenge-score-form-new' challenge.id %}"> Do the challenge </a>
+                        <a class="btn btn-primary pull-right col-xs-3" href="{% url 'challenge-score-form' challenge.id anxiety_score_card.id %}"> Do the challenge </a>
                     </li>
+                    <hr>
                 </ul>
+                {% else %}
+                    {% if challenge_in_progress %}
```

Comment:
Nitpick: can we change this function name to something like `another_challenge_in_progress` for better readability?

Reply:
Yes no problem will send through change now